pkg/controller/utils/unitConversions.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018 VMware Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+// BytesToGB converts from bytes(int64) to GB(float64)
+func BytesToGB(val int64) float64 {
+	return float64(val) / (1024.0 * 1024.0 * 1024.0)
+}
```

Comment:
Good Abstraction 👍

Reply:
Its sreekanth's abstraction in plugin side, just copied that into controller too. :smile: